docs/admin/service_manager.md

```diff
@@ -0,0 +1,189 @@
+endpoint:
+
+* **name:** Name of the stack - this is equivalent to the stack_name parameter
+  used in a docker stack deploy command;
+
+* **compose-file:** A string containing the entire docker-compose file in yaml
+  format; Variables may be referenced in this file which are obtained from
+  compose-vars; The syntax used for variable substitution is analogous to
+  docker-compose, i.e. `${VARIABLE}`.
+
+* **compose-vars:** Also a string in yaml format containing the a key-value
+  mapping of the variables previously defined.
+
+* **engine-url:** Swarm manager docker engine url. For example:
+  `tcp://swarmmanager.zurich.fiwarelab.org:2376`
+
+* **ca-cert:** A string containing the ca.pem certificate protecting the
+  docker-engine.
+
+* **cert:**  A string containing the cert.pem certificate protecting the
+  docker-engine.
+
+* **cert-key:** A string containing the key.pem certificate protecting the
+  docker-engine.
+
+* **external-files:** An array of JSON objects containing all external files
+  required by the compose-file, .i.e secrets and configs. The JSON object
+syntax should be `{‘filename’: ‘file_content’}`.
+
+The IP address of the Swarm manager can be found in the [Rancher](http://rancher.com/)
+interface under the tab Infrastructure -> Hosts (More details below). Note
+that a swarm may have more than one manager: in this case, any of the swarm
+managers could be chosen. The default port used by Docker engine is 2376 and
+this is the port usually used in the engine-url parameter. *Note that
+responsibility for ensuring this port is open on the VM lies with the user -
+it may require specifically setting up security groups and/or other firewall
+rules to ensure these ports are accessible*.
+
+## Security
+
+In order to safely connect to a remote Docker engine, the Mastermind Service
+Manager requires TLS certificates to protect the connection to the remote
+Docker engine. [This is a common practice in Docker](https://docs.docker.com/engine/security/https/).
+
+[Rancher](http://rancher.com/) makes use of docker-machine which automatically
+generates certificates and protects the engine for every host created - note
+that this does not apply for custom hosts, i.e. hosts which are not created
+using docker-machine.
+
+All certificates can be easily retrieved from Rancher’s UI, in the
+Infrastructure -> Hosts page select the host which has the label
+`swarm=manager` and at the top right section there is a button to edit the
+Host, click on it then click on `Machine Config`. This will download a zip file
+containing all certificates and keys that can be used to access the host.
+
+## Docker Compose Reference - Parameters Supported by the Service Manager
+
+The Docker compose file format is quite complex, reflecting the complexity of
+container systems. The Mastermind Service Manager does not currently support
+all parameters defined in docker-compose. Below, the set of supported
+parameters is provided.
+
+### Services
+
+* command:
```

Comment:
if you add these parameters in bold or as code it'll make the list more readable.

Reply:
I have tried adding it in bold but somehow the css is not applying it to lists. Anything we could help to solve this?